src/pages/LookupPublicKeyTestPage.tsx

```diff
@@ -0,0 +1,5 @@
+import { LookupPublicKey } from 'components/lookupPublicKey';
+
+export function LookupPublicKeyTestPage() {
```

Comment:
Good pattern. I like to keep the pages simple and put the logic in the components.